tests/field_test.py

```diff
@@ -1,18 +1,23 @@
-from pyrsistent import field
+from pyrsistent._compat import Enum
```

Comment:
I don't like the idea of using private types and APIs from the tests. It ties the tests to tightly to the internal structure of the package in my opinion.

There's only one place where we do that previously and that is related to covering the C-implementation and Python-implementation of the PVector with the exact same tests.

Reply:
so in this case, I think the following points apply:

1. I'm not actually testing `pyrsistent._compat.Enum`, I just need it for these tests
2. I cannot implement this coverage without using something `Enum`-like
3. I can't use `from enum import Enum` because of compat reasons, so I need a compatible import. I might as well use the one we've already defined, then.

Reply:
No it's not a great tradgedy. :-) An alternative would be to only execute these tests cases for Python 3. If you keep it as is you may want to add a brief comment on the `TestEnum` to state why it derives from the internal Enum class.